src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -14,22 +14,24 @@
 
 import teammates.common.util.Const.SystemParams;
 
-/** A helper class to hold time-related functions (e.g., converting dates to strings etc.).
- * Time zone is assumed as UTC unless specifically mentioned.
+/**
+ * A helper class to hold time-related functions (e.g., converting dates to
+ * strings etc.). Time zone is assumed as UTC unless specifically mentioned.
  */
 public class TimeHelper {
-    
+
     private static final Map<String, String> TIME_ZONE_CITIES_MAP = new HashMap<String, String>();
     private static final List<Double> TIME_ZONE_VALUES = new ArrayList<Double>();
-    
+
     /*
-     *This time zone - city map was created by selecting major cities from each time zone.
-     *reference: http://en.wikipedia.org/wiki/List_of_UTC_time_offsets 
-     *The map was verified by comparing with world clock from http://www.timeanddate.com/worldclock/
-     *Note: No DST is handled here.
+     * This time zone - city map was created by selecting major cities from each
```

Comment:
@Sumei1009, don't refactor code not related to your issue. This makes reviewing your code hard. A pull request should only do one thing.